content/plugins/ipam/dhcp.md

```diff
@@ -24,13 +24,15 @@ $ ./dhcp daemon
 ```
 
 If given `-pidfile <path>` arguments after 'daemon', the dhcp plugin will write
-its PID to the given file.  
+its PID to the given file.
 If given `-hostprefix <prefix>` arguments after 'daemon', the dhcp plugin will
 use this prefix for DHCP socket as `<prefix>/run/cni/dhcp.sock`. You can use
 this prefix for references to the host filesystem, e.g. to access netns and the
-unix socket.  
+unix socket.
```

Comment:
```suggestion
unix socket.  
```
Same trick as before, adding 2 spaces will ensure that the next "If given" will appear on the next line.